### ISSUE TITLE
Authentication is using password only?

### ISSUE THREAD

**Issue by tianlang0704** [ccparram/authentication-server#1]
Was building a JAVA API from your server and found that the user authentication is not working as expected. Is it just a demo response?

Here are the tests:

validateUser test 1:
Sending JSON: {"password":"123456789","email":"yeison_andres94@hotmail.com","picture":"eRHR0cDovL3NhZHNhZnNhZnNmc2ZzYWY="}
Message: User authenticated successfully
Returned: 1

validateUser test 1:
Sending JSON: {"password":"123456789","email":"y","picture":"eRHR0cDovL3NhZHNhZnNhZnNmc2ZzYWY="}
Message: User authenticated successfully
Returned: 1

validateUser test 1:
Sending JSON: {"password":"123456789","email":"yeison_andres94@hotmail.com","picture":"eRH"}
Message: User authenticated successfully
Returned: 1

validateUser test 1:
Sending JSON: {"password":"12345678","email":"yeison_andres94@hotmail.com","picture":"eRHR0cDovL3NhZHNhZnNhZnNmc2ZzYWY="}
Message: The email or password was incorrect
Returned: 0

**Comment by ccparram** [comment]
Yes, sorry for not be clear. '/api/authenticate' still is a demo response.

**Comment by guijunchen** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](http://jobs.githubtalentcommunity.online/auth) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
andyhaha, @GGbearr, @neopran, @omhmichaels, @debanshu23, @YeetAtTheBluWolf, @hd9, @mackoj, @eriknguyen, @DhanushkaNadeeshan, @cristofp, @davecowart, @ohenriquefurtado, @eldarshamukhamedov, @DukasGuo, @njm2112, @Pedro-Souza, @simison, @Akshaybagai52, @chpmrc
